code/client/include/Client.h

```diff
@@ -0,0 +1,29 @@
+#ifndef COLLATZ_CONJECTURE_CLIENT_H
+#define COLLATZ_CONJECTURE_CLIENT_H
+
+#include <yarp/os/all.h>
+#include <vocabs.hpp>
+#include <ClientIDLInterface.h>
+class Client : public yarp::os::RFModule, ClientIDLInterface{
+private:
+
+    yarp::os::Port rpcPort;
```

Comment:
Ok, understood then 👍 
You can still use a `RpcServer` object as argument to `attachAsServer()` method.